pyesgf/logon.py

```diff
@@ -82,10 +82,18 @@ class LogonManager(object):
     STATE_EXPIRED_CREDENTIALS = 2
     STATE_INVALID_CREDENTIALS = 3
 
-    def __init__(self, esgf_dir=ESGF_DIR, dap_config=DAP_CONFIG):
+    def __init__(self, esgf_dir=ESGF_DIR, dap_config=DAP_CONFIG,
+                 ssl_verify=True):
```

Comment:
Use ``verify`` instead of ``ssl_verify`` to conform with the ``SearchConnection`` and ``requests``.